tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1143,6 +1169,131 @@ def test_login_failure_rhsso_user_if_internal_user_exist(
         assert error.typename == "NavigationTriesExceeded"
 
 
+@destructive
+def test_user_permissions_rhsso_user_after_group_delete(
+    enable_external_auth_rhsso, rhsso_setting_setup, session, module_org, module_loc,
+):
+    """Verify the rhsso user permissions in satellite should get revoked after the
+        termination of rhsso user's external rhsso group
+
+    :id: 782926c0-d109-41a0-af7a-bffd658f59d7
+
+    :CaseImportance: Medium
+
+    :steps:
+        1. create usergroup with admin permissions respectively
+        2. assigned that group to rhsso user
+        3. verify the permission of the rhsso user in Satellite
+        4. delete the rhsso group
+
+    :expectedresults: external rhsso user's permissions should get revoked after external rhsso
+        group deletion.
```

Comment:
How does that work for user that is already logged in? Will log him out / revoke permissions on next action, or only on next login after session timeout?

That would be stronger assertion and worth checking, too.

Reply:
This is after session timeout and logout I have tested, I have already test case around timeout. I will once again cross check if it on the browser refresh. If yes will update the test case, otherwise I don't see any change needs to be made

Reply:
@mirekdlugosz I think whatever I have added it make sense. We already have timeout test case, if we include this then the test execution time will increase. We can not use logout here because in RHSSO there nothing called hard logout. If you still have query we can discuss this offline.